src/engine/Sprite.cxx

```
@@ -30,7 +30,18 @@ void Sprite::render() const
     {
       if (highlightSprite)
       {
-        SDL_SetTextureColorMod(m_SpriteData[currentLayer].texture, highlightColor.r, highlightColor.g, highlightColor.b);
+        if (SDL_SetTextureColorMod(m_SpriteData[currentLayer].texture, highlightColor.r, highlightColor.g, highlightColor.b))
+        {
+          // indicates failure.
+        }
+      }
+
+      if (transparentSprite)
+      {
+        if (SDL_SetTextureAlphaMod(m_SpriteData[currentLayer].texture, alpha))
+        {
+          // indicates failure.
```

Put at least TODO, if error is not handled